pages/contact/age/index.js

```diff
@@ -0,0 +1,21 @@
+import ContactNavigation from "../../../components/layout/ContactNavigation";
+
+import classes from "../../../styles/aboutUs.module.css";
+
+function age() {
```

Comment:
By convention, React components are usually named using `PascalCase` (as opposed to `camelCase` or `snake_case`).

A better name for this component would be `AgePage` or something like that.

Because it's exported as the default export, the name does not really matter anywhere in the code (because the name can be defined by the user of the component when importing it, `import MyOwnName` from 'path/to/component'). But the name is still used in some places, especially in debugging and editor code completion.